great_expectations/rule_based_profiler/data_assistant/data_assistant_dispatcher.py

```diff
@@ -61,3 +61,54 @@ def __getattr__(self, name) -> DataAssistantRunner:
             ] = data_assistant_runner
 
         return data_assistant_runner
+
+    @staticmethod
```

Comment:
Should these be class methods instead? Always get tripped up between the two.

Reply:
I'd use class methods since you invoke other class methods (and can specify `cls._registered_data_assistants` lower down).

Reply:
perfect!